.github/workflows/build-test.yml

```diff
@@ -338,7 +338,7 @@ jobs:
         with:
             toolchain: stable
       - name: cargo test
-        run: cargo test
+        run: cargo test --workspace --all-features
```

Comment:
there's still a problem with the pyo3 wheel failing when running tests (and benchmarks I believe). I haven't figured out a way around this yet. Assistance would be appreciated.

Reply:
I ran `git bisect` and the change that breaks this is 625e51fc63c0ea1082a4b1f0d5a1a85d343841e0.

Reply:
See also https://github.com/PyO3/pyo3/issues/340 and https://github.com/PyO3/pyo3/issues/341

Reply:
Incidentally, https://pyo3.rs/main/features.html#multiple-pymethods suggests `multiple-pymethods` may break wasm builds. :(